### ISSUE TITLE
how do you personally use this?

### ISSUE THREAD

**Issue by leshy** [chjj/pulsemixer.js#1]
Hey, I went through your repos and they are awesome! nmterm, pulsemixer, termcoin, all those things I wanted, but never took time to write. thank you!

I really like slock fork also, already using it.

I know I can write a small app.js and call start() on your lib, but I see that these things are not written as apps but libs, so I'm curious about how you personally use them in your system? do you have some kind of a blog or something, I'm curious about your setup in general.
thanks

**Comment by chjj** [comment]
They're meant to be apps, terminal apps.

nmterm and pulsemixer are incomplete at the moment. I wanted to create a wicd-curses version of NetworkManagerEditor, and an alsamixer version of pavucontrol.

Right now, nmterm is probably the furthest along, but it's still not developed enough to add and edit networks.

So, these projects are very immature and unfinished, but I want to finish them sometime in the future.

I'm glad you like the slock fork. Feel free to check out the code for nmterm and see if you can get some add/edit network features in.

**Comment by chjj** [comment]
Also, termcoin is a different story. termcoin is actually a complete terminal app. Obviously it could still use some improvements and revisions as every app could, but it's usable in production. nmterm and pulsemixer were just two projects I started because I switched from wicd to NetworkManager and ALSA to PulseAudio and wanted terminal apps to control them.

I used to have a blog. I might put it back up soon.

**Comment by leshy** [comment]
Yeah I understand the motives, never saw anything as fast and simple as alsamixer for pulseaudio, and control of networkmanager has always been a torn in the eye too. Never took time to work on my own apps for these things, I just have a bunch of workarounds.

ok, thanks for the clarification. thought you might be using these things somehow through your term.js or something.

PS
of course, I'll keep in mind to do pull requests if I end up working on nmterm.
